life_expectancy/cleaning.py

```diff
@@ -0,0 +1,67 @@
+import os
+import numpy as np
+import pandas as pd
+import argparse
+
+
+# Determines the absolute path of the directory we are working on 
+file_dir = os.path.dirname(os.path.abspath(__file__))
+
+# Determines the relative path of the directory we are working on
+data_dir = os.path.join(file_dir, 'data')
+
+
+def split_column(df_: pd.DataFrame) -> pd.DataFrame:
+    """splits a column into several, based on a comma separating diff values"""
+    df = df_.copy()
+    new_cols = df['unit,sex,age,geo\\time'].str.split(',', expand=True)
+    new_cols.columns = ['unit', 'sex', 'age', 'region']
+    df = pd.concat([df, new_cols], axis=1)
+    df = df.drop(['unit,sex,age,geo\\time'], axis=1)
+    return df
```

Comment:
It is just my thought but I think it is a good idea to parse the columns name as arguments instead of hard-coding them. This way we can reuse the function for any other tsv files. But again, this is just a suggestion 😄